src/main/resources/db/migration/V1__init_tables.sql

```diff
@@ -12,14 +12,30 @@ CREATE TABLE organisation
 
 CREATE TABLE professional_user
 (
-  ID              UUID,
-  FIRST_NAME      VARCHAR(255) NOT NULL,
-  LAST_NAME       VARCHAR(255) NOT NULL,
-  EMAIL_ADDRESS   VARCHAR(255) NOT NULL,
-  STATUS          VARCHAR(50) NOT NULL,
-  ORGANISATION_ID UUID NOT NULL,
-  LAST_UPDATED    TIMESTAMP,
-  CREATED         TIMESTAMP NOT NULL,
+  ID                 UUID,
+  FIRST_NAME         VARCHAR(255) NOT NULL,
+  LAST_NAME          VARCHAR(255) NOT NULL,
+  EMAIL_ADDRESS      VARCHAR(255) NOT NULL,
+  STATUS             VARCHAR(50) NOT NULL,
+  ORGANISATION_ID    UUID NOT NULL,
+  PAYMENT_ACCOUNT_ID UUID,
+  LAST_UPDATED       TIMESTAMP,
+  CREATED            TIMESTAMP NOT NULL,
   PRIMARY KEY (ID),
   FOREIGN KEY (ORGANISATION_ID) REFERENCES organisation (id)
 );
+
+CREATE TABLE payment_account
+(
+  ID                   UUID,
+  PBA_NUMBER           VARCHAR(255) NOT NULL UNIQUE,
+  ORGANISATION_ID      UUID,
+  PROFESSIONAL_USER_ID UUID,
+  LAST_UPDATED         TIMESTAMP NOT NULL,
+  CREATED              TIMESTAMP NOT NULL,
+  PRIMARY KEY (ID),
+  FOREIGN KEY (ORGANISATION_ID) REFERENCES organisation (id),
+  FOREIGN KEY (PROFESSIONAL_USER_ID) REFERENCES professional_user (id)
+);
+
+ALTER TABLE professional_user ADD FOREIGN KEY (PAYMENT_ACCOUNT_ID) REFERENCES payment_account (id);
```

Comment:
We have removed the foreign key from the data model please revise.
You should not be mixing code changes with infrastructure. Is it possible to revert this file without breaking your S2S vault fix.